OctaneSdk/Connector/Authentication/LwssoAuthenticationStrategy.cs

```diff
@@ -18,14 +31,14 @@ public class LwssoAuthenticationStrategy : AuthenticationStrategy
         private static string LWSSO_COOKIE_NAME = "LWSSO_COOKIE_KEY";
         private static string OCTANE_USER_COOKIE_NAME = "OCTANE_USER";
 
-        private ConnectionInfo credentials;
+        private UserPassConnectionInfo credentials;
```

Comment:
credentials might be also APIKeyConnectionInfo. Please revert it to ConnectionInfo